### DISCUSSION TITLE
Will you, in the future, have plans to making stable pocketbase release a Freemium?

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [pocketbase/pocketbase#2841] [Q&A] [answered]
If that is the case, can we still use the unstable version (i.e. less than v1.0.0) for free? Right now, in its current version, I almost can do all what I want to do and what I needed, and I'm very happy on it as it makes the development of my meta-framework faster.

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase will always remain open source and there are no plans to change the MIT license even when we reach a stable v1.

**Reply by mrbu-solutions** [reply]
Thank you very much if that is the case.
